operator/internal/central/extensions/reconcile_pvc_test.go

```diff
@@ -81,71 +81,40 @@ func notOwnedBy(central *platform.Central) pvcVerifyFunc {
 }
 
 func TestReconcilePVCExtension(t *testing.T) {
-	removedCentral := makeCentral(nil)
-	emptyNotDeletedCentral := makeCentral(nil)
-	emptyNotDeletedCentralWithDB := makeCentral(nil)
-	emptyNotDeletedCentralWithDB.Spec.Central.DB = &platform.CentralDBSpec{}
+	emptyCentral := makeCentral(nil)
+	emptyCentralWithDB := makeCentral(nil)
+	emptyCentralWithDB.Spec.Central.DB = &platform.CentralDBSpec{}
 
 	pvcObsoletedAnnotation := map[string]string{
 		common.CentralPVCObsoleteAnnotation: "true",
 	}
-	centralWithPvcObsoletedAnnotation := makeCentral(nil)
-	centralWithPvcObsoletedAnnotation.Annotations = pvcObsoletedAnnotation
-	centralWithPersistenceAndPvcObsoletedAnnotation := makeCentral(&platform.Persistence{
-		PersistentVolumeClaim: &platform.PersistentVolumeClaim{
-			Size:             pointer.String("500Gi"),
-			StorageClassName: pointer.String("new-storage-class"),
-			ClaimName:        pointer.String(testPVCName),
-		},
-	})
-	centralWithPersistenceAndPvcObsoletedAnnotation.Annotations = pvcObsoletedAnnotation
+	emptyCentralWithPvcObsoletedAnnotation := makeCentral(nil)
+	emptyCentralWithPvcObsoletedAnnotation.Annotations = pvcObsoletedAnnotation
 
 	externalCentralWithDB := makeCentral(nil)
 	externalCentralWithDB.Spec.Central.DB = &platform.CentralDBSpec{}
```

Comment:
I think this line is not needed now either, that `makeCentral` initializes the DB struct...